src/Connector.sol

```diff
@@ -111,7 +122,7 @@ contract CentrifugeConnector is Test {
 
         address memberlist = memberlistFactory.newMemberlist();
         RestrictedTokenLike(token).depend("memberlist", memberlist);
-
+        MemberlistLike(memberlist).updateMember(address(this), uint(-1)); // required to be able to receive tokens in case of withdrawals   
```

Comment:
I am a bit confused about this call; the function definition I find is this below so how are we passing the address and a negative `validUntil`? Also, does `-1` `validUntil` means we set it as valid indefinitely? 

```solidity
uint64 poolId,
bytes16 trancheId,
address user,
uint256 validUntil
```

Reply:
`-1` is basically a max integer, so whitelisting for an infinite period. @AStox you can also do `type(uint256).max` which is more readable.

I'm not sure where you found the function definition. If you look at the `MemberlistLike` interface, the interface is
```
function updateMember(address usr, uint validUntil) external
```

The pool id and tranche id aren't needed because a memberlist contract is deployed for each specific token, so it's already implicit.

Reply:
> The pool id and tranche id aren't needed because a memberlist contract is deployed for each specific token, so it's already implicit.

I see. The definition I was referring to is defined in `src/Connector.sol` but I get it that these are at different levels of abstraction 👍